README.md

```diff
@@ -11,12 +11,17 @@ information.
 
 ## Version support
 
-| Version  | Support                            |
-|----------|------------------------------------|
-|    3.1.x | Bug fixes and security patches.    |
-|    3.0.x | Security patches only.             |
-|    2.2.x | Security patches only.             |
-| <= 2.1.x | End of support.                    |
+| Version  | Support                            | First Released     | End of Support |
+|----------|------------------------------------|--------------------|----------------|
+|    3.2.x | Bug fixes and security patches.    | 2025-07-30         | 2028-12-31     |
+|    3.1.x | Security patches only.             | 2024-06-11         | 2027-12-31     |
+|    3.0.x | Security patches only.             | 2022-09-06         | 2026-12-31     |
```

Comment:
Almost nobody is running 3.0, so I'd be OK dropping 3.0 support now, though I'm also fine keeping it. Considering 2.2 usage, we should probably keep security support longer than the end of this year, IMO.I don't think it makes sense to keep security patches for 3.0 but drop security patches for 2.2, when 2.2 is far more popular.